src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -41,48 +41,27 @@ module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
  */
 /* jshint latedef: nofunc */
 function dialogTestHelper(args) {
-    // Sinon doesn't allow wrapping of `window.open` with a spy because `open`
-    // doesn't look like a function in these browsers.
-    var sinonCantWrapWindowOpen = jQuery.browser.msie &&
-        jQuery.browser.versionNumber <= 8;
+    var wymeditor = jQuery.wymeditors(0);
```

Comment:
How about `tryDialogWithClickAndKeyboardThenAssert`? Long name, but this function does a lot.

Mostly, it's very unclear from reading these tests right now that we're testing both buttons and toolbar buttons. The docstring could also be updated to say that "...window was opened or not via both a toolbar click and a keyboard shortcut".